itkcuberille/itk_wasm_env.bash

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+function die() {
+    echo "$1"
+    exit 1
+}
+
+export ITK_WASM_TEST_DATA_HASH=${ITK_WASM_TEST_DATA_HASH:-$(cat package.json | jq -e -r '."itk-wasm"."test-data-hash"')}
+export ITK_WASM_TEST_DATA_URLS=${ITK_WASM_TEST_DATA_URLS:-$(cat package.json | jq -e -r '."itk-wasm"."test-data-urls" | join(" ")')}
+
+export ITK_WASM_ITK_REPOSITORY=${ITK_WASM_ITK_REPOSITORY:-"https://github.com/thewtex/ITK"}
```

Comment:
Is this permanently expected to point to your fork?

Reply:
This is expected to point to my fork until the patches have been merged upstream.